amplify/backend/auth/covidcounterba0a32fb/parameters.json

```diff
@@ -45,16 +45,12 @@
             "Arn"
         ]
     },
-    "useDefault": "defaultSocial",
```

Comment:
This should stay as defaultSocial i think

Reply:
sure Shannon I will add defaultSocial instead of default